jsonrpc/server.go

```diff
@@ -17,11 +17,12 @@ import (
 )
 
 const (
-	InvalidJSON    = -32700 // Invalid JSON was received by the server.
-	InvalidRequest = -32600 // The JSON sent is not a valid Request object.
-	MethodNotFound = -32601 // The method does not exist / is not available.
-	InvalidParams  = -32602 // Invalid method parameter(s).
-	InternalError  = -32603 // Internal JSON-RPC error.
+	InvalidJSON         = -32700 // Invalid JSON was received by the server.
+	InvalidRequest      = -32600 // The JSON sent is not a valid Request object.
+	MethodNotFound      = -32601 // The method does not exist / is not available.
+	InvalidParams       = -32602 // Invalid method parameter(s).
+	InternalError       = -32603 // Internal JSON-RPC error.
+	TransactionNotFound = -32004 // Transaction JSON-RPC error.
```

Comment:
The issue is specific to the Starknet spec and not applicable for all jsonrpc use cases. It should be solved in the [rpc package](https://github.com/NethermindEth/juno/blob/8cf2dff7f2fc2561f4fe5b9cbf863da033d00453/rpc/handlers.go#L31C2-L31C2) and not the `jsonrpc` package.